web/impact/impact/tests/test_user_history_view.py

```diff
@@ -166,10 +166,10 @@ def test_user_became_confirmed_judge(self):
             self.assertEqual(1, len(events))
             format_string = UserBecameConfirmedJudgeEvent.PROGRAM_ROLE_FORMAT
             self.assertEqual(format_string.format(
-                    role_name=UserBecameConfirmedJudgeEvent.ROLE_NAME,
-                    name=prg.program_role.name,
-                    id=prg.program_role.id),
-                             events[0]["description"])
+                role_name=UserBecameConfirmedJudgeEvent.ROLE_NAME,
```

Comment:
Not a blocker, but I'm not thrill by this change since it seems harder to parse.  pycodestyle is cool with either (hence not a blocker), but in the new way it isn't clear to me that some of the args are format and some are for assertEqual.  I suspect this is an editor thing since emacs wants the other indent and maybe whatever you're using wants it this way.

Reply:
I think I reverted it. Either way it's an auto-indent thing, which I relied heavily on in this PR, naturally.